### DISCUSSION TITLE
Release dependencies as open source

### DISCUSSION THREAD

**Discussion by jimmarino** [International-Data-Spaces-Association/DataspaceConnector#55] [General]
Hi,

It appears there are connector dependencies that are not open source projects, even though some of them may be released to Maven central under the ASL 2.0 license.

It would be great from a community perspective (bug reporting, transparent development, roadmap, patches, etc.) if the following connector dependencies could be developed as open source: 

de.fraunhofer.iais.eis.ids:interaction:4.0.2-SNAPSHOT
de.fraunhofer.iais.eis.ids:protocol:4.0.2-SNAPSHOT
de.fraunhofer.iais.eis.ids:ecosystemintegration:4.0.2-SNAPSHOT
de.fraunhofer.iais.eis.ids:core:4.0.2-SNAPSHOT
de.fraunhofer.iais.eis.ids:connector-commons:4.0.2-SNAPSHOT
de.fraunhofer.iais.eis.ids.infomodel:java:4.0.0
de.fraunhofer.iais.eis.ids.infomodel:validation-serialization-provider:4.0.0
de.fraunhofer.iais.eis.ids:infomodel-serializer:4.0.0

Thanks!

**Comment by sebbader** [comment]
Hello @jimmarino!

That is a really relevant topic, which we already have on our list and are working on. For now, these projects are not open-source-ready, neither in regards to documentation, community-building, licenses etc. 

There is however a tutorial that can help you to better understand the usage of these libraries: https://jira.iais.fraunhofer.de/stash/projects/ICTSL/repos/ids-infomodel-demo/browse/

@DennisKubitza, @maboeckmann: can you please keep us updated here when we have progress to announce?

Thanks!
Sebastian
